ipc/provider/src/manager/evm/manager.rs

```diff
@@ -14,7 +14,7 @@ use ipc_actors_abis::{
     subnet_actor_reward_facet,
 };
 use ipc_api::evm::{fil_to_eth_amount, payload_to_evm_address, subnet_id_to_evm_addresses};
-use ipc_api::validator::from_contract_validators;
+use ipc_api::validator::vec_try_from;
```

Comment:
Since this is a public and completely generic function, just a convenience for some library builtins, it would be worth moving it into a utility module. It's strange to see it being imported from `validator` when it has nothing validator specific about it.

Reply:
I don't even see it used anywhere else, so maybe just move it into this module as a private function.